apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -94,7 +94,7 @@ AprilTagDetector::AprilTagDetector(ros::NodeHandle& nh, ros::NodeHandle& pnh) :
     tag_codes = &AprilTags::tagCodes36h11;
   }
 
-  pnh.param<bool>("start_enabled", enabled_, false);
+  pnh.param<int>("start_enabled", number_of_frames_to_capture_, 0);
```

Comment:
This should probably not still be called 'start_enabled'

Reply:
The purpose of this is still to determine whether we are enabled on start or not, irrespective of what is the variable we use.

Reply:
After discussing this, I think it's fine.